config/fxdata/magic.cfg

```diff
@@ -462,7 +463,7 @@ Animation = 0
 AnimationSize = 0
 AnimationTransparency = 0
 Health = 10
-Damage = 4
+Damage = 3
```

Comment:
why was the flame breath nerfed? does this have anything to do with rest of issue?

Reply:
In the original game it did 3 damage. But due to stat boost fixes, the dragons became a lot weaker, including falling behind in damage, so damage was boosted from 3 to 4.

In KeeperFX, spell damage grows with XP, but I found that some spells are excluded in this. This PR changes it from an hardcoded change to a spell property. Flame Breath was one of the spells that was excluded by growth damage.
In short:
1) KeeperFX had everybody do more damage, but flame breath was excluded
2) Flame Breath got a stat boost to compensate

So now on this PR:

3) Flame breath now grows with level too
4) The stat boost is reverted as it is no longer needed